qiskit/_quantumprogram.py

```diff
@@ -889,6 +889,48 @@ def compile(self, name_of_circuits, backend="local_qasm_simulator",
             qobj["circuits"].append(job)
         return qobj
 
+    def reconfig(self, qobj, backend=None, config=None, shots=None, max_credits=None, seed=None):
+        """Change configuration parameters for a compile qobj. Only parameters which
+        don't affect the circuit compilation can change, e.g., the coupling_map
+        cannot be changed here!
+
+        Notes:
+            If the inputs are left as None then the qobj is not updated
+
+        Args:
+            qobj (dict): already compile qobj
+            backend (str): see .compile
+            config (dict): see .compile
+            shots (int): see .compile
+            max_credits (int): see .compile
+            seed (int): see .compile
+
+        Returns:
+            qobj: updated qobj
+        """
+
+        if backend is not None:
+            qobj['config']['backend'] = backend
+
+        if shots is not None:
+            qobj['config']['shots'] = shots
+
+        if max_credits is not None:
+            qobj['config']['max_credits'] = max_credits
+
+
+        for circuits in qobj['circuits']:
+
+            if seed is not None:
+                circuits['seed'] = seed
+
+            if config is not None:
+                circuits['config'].update(config)
```

Comment:
What if I:
```python
q_program.config( backend='ibmqx5', shots=1024, config=_my_config_for_backend_ibmqx3_and_shots_4096)
```
config parameter will always override the rest of named parameters.
Should we move the check of the config parameter to the beginning of the function, and state in the docstring that named parameters will have preference over the ones in config?

Reply:
This config is something that gets applied to the circuits in qobj, so won't override these other options (if you look at .compile())

Reply:
Yep! You are right.. guess I misread the code